src/apps/material/helper.ts

```diff
@@ -350,3 +350,12 @@ export const getBestMaterialTypeForWork = (work: Work) => {
 export const reservationModalId = (faustIds: FaustId[]) => {
   return constructModalId("reservation-modal", faustIds.sort());
 };
+
+export const getDbcVerifiedSubjectsFirst = (subjects: Work["subjects"]) =>
+  Array.from(
+    new Set([
```

Comment:
Consider using `_.uniq` instead of combining `Array.form` + `Set`. We have Lodash in our toolchain anyway and `_.uniq` makes your intention more readable.

Also consider adding a comment explaining why we need remove duplicates.